docs/components/arm/lite6.md

```diff
@@ -102,6 +102,41 @@ See [the frame system service](/services/frame-system/) for more information on
 
 {{< readfile "/static/include/components/test-control/arm-control.md" >}}
 
+### DoCommand
+
+Execute the model-specific commands `set_speed` and `set_acceleration` that are not otherwise defined by the component API.
```

Comment:
This doesn't explain what this is or why it's here.

```suggestion
### Additional commands

In addition to the [Arm API](/appendix/apis/components/arm/), the `lite6` arm supports some model-specific commands that allow you to set the speed and the acceleration of the arm.

You can invoke these commands by passing the following JSON document to the [`DoCommand()`](/appendix/apis/components/arm/#docommand) method:

```json
{
    "set_speed": 45.0, // Set speed to 45.0 degrees per second
    "set_acceleration": 10.0  // Set acceleration to 10.0 degrees per second per second
}
\```

TODO(Add table of possible arguments)

```